static/app/views/insights/browser/webVitals/queries/useSpanSamplesWebVitalsQuery.tsx

```diff
@@ -68,9 +74,9 @@ export function useSpanSamplesWebVitalsQuery({
         SpanIndexedField.INP,
         SpanIndexedField.LCP,
         SpanIndexedField.CLS,
-        SpanIndexedField.INP_SCORE,
-        SpanIndexedField.LCP_SCORE,
-        SpanIndexedField.CLS_SCORE,
+        SpanIndexedField.INP_SCORE_RATIO,
+        SpanIndexedField.LCP_SCORE_RATIO,
+        SpanIndexedField.CLS_SCORE_RATIO,
```

Comment:
We switch to using score ratio measurements so we don't have to fetch two measurements and do score/weight to calculate the score ratio